doc/getting-started.md

```diff
@@ -84,4 +84,15 @@ If you want to remove the version `1.0.0` from awesome-project.
 
 ```sh
 curl -X DELETE --header "Docat-Api-Key: <token>" http://localhost:8000/api/awesome-project/1.0.0
+```
+
+#### Upload Project Icon
+
+To upload a icon (only jpegs are supported), you don't need a token, except if 
```

Comment:
maybe add an issue that we should support more formats in the future?

Reply:
idk lets keep it simple and just allow one file format should be enough? but then maybe we should go for png as the only supported file format?

Reply:
Exactly, I don't see a good reason why png for example is not supported

Reply:
I just used what was there, the project repository looks like this so I fitted it to what was there:

```
  /**
   * Returns the logo URL of a given project
   * @param {string} projectName Name of the project
   */
  getProjectLogoURL(projectName) {
    return `${this.baseURL}/${resource}/${projectName}/logo.jpg`
  },

```

Reply:
Ok, I changed it to PNGs